tests/copy-paste.js

```diff
@@ -697,6 +697,44 @@ require([
             assert(util.getMug("select/itemset"), "itemset should not be cut");
         });
 
+        it("should copy dynamic select with itemset data and instance", function () {
+            util.loadXML("");
+            paste([
+                ["id", "type", "labelItext:en-default", "labelItext:hin-default", "instances", "itemsetData"],
```

Comment:
This goes for others in this file as well (I realize they aren't all the same), but why not declare these once `var data= [['id',...], [...],...]` and use it like `paste[data]` `eq(mod.copy(), data)` ?